src/main/java/view/OutputView.java

```diff
@@ -7,21 +7,25 @@
 
 public class OutputView {
 
-    public static String POSITION_MARK = "-";
+    private static final String POSITION_MARK = "-";
 
-    public void printEachRound(List<CarStatusDto> carStatusDtos) {
-        carStatusDtos.forEach(carStatusDto -> printEachCarStatus(carStatusDto));
-        System.out.println("");
+    public void printGameResult(List<List<CarStatusDto>> result) {
+        result.forEach(this::printEachRoundResult);
     }
 
-    public void printWinners(WinnersNameDto winnersNameDto) {
-        System.out.println(getFormattedName(winnersNameDto.getWinnersNames()) + "가 최종 우승했습니다.");
+    private void printEachRoundResult(List<CarStatusDto> carStatusDtos) {
+        carStatusDtos.forEach(this::printEachCarsPosition);
+        System.out.println("");
     }
 
-    private void printEachCarStatus(CarStatusDto carStatusDto) {
+    private void printEachCarsPosition(CarStatusDto carStatusDto) {
```

Comment:
```suggestion
    private void printEachCarPosition(CarStatusDto carStatusDto) {
```

실수로 복수형으로 쓰신 것 같네요 ㅎㅎ